openmc_models/plot_isos.py

```diff
@@ -0,0 +1,347 @@
+import openmc
+import openmc.deplete
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+
+
+
+def plot_isos(title, ylabel, xlabel, fig_fname, dep_fname, mat_id, 
+              nuc_list=['U235', 'U236', 
+                        'Pu238', 'Pu239', 'Pu240', 'Pu241', 
+                        'Cs134', 'Cs137'], 
```

Comment:
Please make this list available to all the functions in this script, either as a function that returns the list or as a (not super recommended, but not terrible) global variable. You repeat this list of strings at least thrice in this one file.